great_expectations/execution_engine/execution_engine.py

```diff
@@ -189,6 +200,27 @@ class SplitDomainKwargs:
 
 
 class ExecutionEngine(ABC):
+    """ExecutionEngine defines interfaces and provides common methods for loading Batch of data and compute metrics.
+
+    ExecutionEngine is the parent class of every backend-specific computational class, tasked with loadsing Batch of
```

Comment:
sorry for the nit - typo
```suggestion
    ExecutionEngine is the parent class of every backend-specific computational class, tasked with loading Batch of
```